indico/migrations/versions/20191002_1820_2496c4adc7e9_add_track_principal.py

```diff
@@ -0,0 +1,231 @@
+            ['event_role_id'],
+            [u'events.roles.id'],
+            name=op.f('fk_track_principals_event_role_id_roles'),
+        ),
+        sa.ForeignKeyConstraint(
+            ['local_group_id'],
+            [u'users.groups.id'],
+            name=op.f('fk_track_principals_local_group_id_groups'),
+        ),
+        sa.ForeignKeyConstraint(
+            ['track_id'],
+            [u'events.tracks.id'],
+            name=op.f('fk_track_principals_track_id_tracks'),
+        ),
+        sa.ForeignKeyConstraint(
+            ['user_id'],
+            [u'users.users.id'],
+            name=op.f('fk_track_principals_user_id_users'),
+        ),
+        sa.PrimaryKeyConstraint('id', name=op.f('pk_track_principals')),
+        schema='events',
+    )
+    op.create_index(
+        op.f('ix_track_principals_email'),
+        'track_principals',
+        ['email'],
+        unique=False,
+        schema='events',
+    )
+    op.create_index(
+        op.f('ix_track_principals_event_role_id'),
+        'track_principals',
+        ['event_role_id'],
+        unique=False,
+        schema='events',
+    )
+    op.create_index(
+        op.f('ix_track_principals_local_group_id'),
+        'track_principals',
+        ['local_group_id'],
+        unique=False,
+        schema='events',
+    )
+    op.create_index(
+        op.f('ix_track_principals_mp_group_provider_mp_group_name'),
+        'track_principals',
+        ['mp_group_provider', 'mp_group_name'],
+        unique=False,
+        schema='events',
+    )
+    op.create_index(
+        op.f('ix_track_principals_track_id'),
+        'track_principals',
+        ['track_id'],
+        unique=False,
+        schema='events',
+    )
+    op.create_index(
+        op.f('ix_track_principals_user_id'),
+        'track_principals',
+        ['user_id'],
+        unique=False,
+        schema='events',
+    )
+    op.create_index(
+        'ix_uq_track_principals_email',
+        'track_principals',
+        ['email', 'track_id'],
+        unique=True,
+        schema='events',
+        postgresql_where=sa.text(u'type = 4'),
+    )
+    op.create_index(
+        'ix_uq_track_principals_local_group',
+        'track_principals',
+        ['local_group_id', 'track_id'],
+        unique=True,
+        schema='events',
+        postgresql_where=sa.text(u'type = 2'),
+    )
+    op.create_index(
+        'ix_uq_track_principals_mp_group',
+        'track_principals',
+        ['mp_group_provider', 'mp_group_name', 'track_id'],
+        unique=True,
+        schema='events',
+        postgresql_where=sa.text(u'type = 3'),
+    )
+    op.create_index(
+        'ix_uq_track_principals_user',
+        'track_principals',
+        ['user_id', 'track_id'],
+        unique=True,
+        schema='events',
+        postgresql_where=sa.text(u'type = 1'),
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index(
```

Comment:
none of those commands are needed since dropping a table will drop everything associated with it